src/content/en/news/status-update17.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Status Update"
+draft: false
+slug: "status-update"
+date: 2021-12-31
+type: "page"
+---
+
+We have processed a total of 209 submissions so far, with 19 of them being eligible for a payout.
+
+We have one submission that is not public yet. Although it is only a note level bug and limited to non default configurations. It will only be revealed once a large % of users update to the up coming 1.7 version. 
```

Comment:
```suggestion
We have one submission that is not public yet. Although it is only a note level bug and limited to non default configurations. It will only be revealed once a large % of users update to the upcoming 1.7 version. 
```